services/gitlab/gitlab-pipeline-coverage.service.js

```diff
@@ -37,50 +38,52 @@ Also make sure you have set up code covrage parsing as described <a href="https:
 </p>
 `
 
-export default class GitlabCoverage extends BaseSvgScrapingService {
+export default class GitlabPipelineCoverage extends BaseSvgScrapingService {
   static category = 'coverage'
 
   static route = {
-    base: 'gitlab/coverage',
-    pattern: ':user/:repo/:branch',
+    base: 'gitlab/pipeline-coverage',
+    pattern: ':project+',
     queryParamSchema,
   }
 
   static examples = [
     {
       title: 'Gitlab code coverage',
-      namedParams: {
-        user: 'gitlab-org',
-        repo: 'gitlab-runner',
-        branch: 'master',
-      },
+      namedParams: { project: 'gitlab-org/gitlab-runner' },
+      queryParams: { branch: 'master' },
+      staticPreview: this.render({ coverage: 67 }),
+      documentation: documentation + moreDocs,
+    },
+    {
+      title:
+        'Gitlab code coverage (with a subgroup in the name of the project)',
+      namedParams: { project: 'megabyte-labs/go/cli/bodega' },
```

Comment:
Nice. One more minor thing. We can drop this example. For all the other GitLab badges (pipeline status, issues, merge requests, releases, etc) they work for repos in nested subgroups but we don't explicitly provide an example in the frontend. In general we try and keep the number of example of different variations in the examples to a minimum if possible. Cheers

Reply:
Ok, example removed ;)